.changeset/five-sloths-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@solid-design-system/components': patch
+---
+
+Update components to reflect properties so that they are always displayed in the DOM. This is specifically required when web components are used within React 19 but also to make dx more predictable.
```

Comment:
```suggestion
Update components to reflect selected properties so that they are always displayed in the DOM. This is specifically required when web components are used within React 19 but also to make dx more predictable.
```